### PR TITLE
Model 0 Artifact

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: data artifact
- *JIRA issue*: [MIC-2895](https://jira.ihme.washington.edu/browse/MIC-2895) 
- *Research reference*: [IV Iron - Children under five](https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_iv_iron/child_model/concept_model.html)

### Verification and Testing
Created artifacts for South Asia and Sub-Saharan Africa and checked that population structure, demographic dimensions, etc looked good.

### CODE REVIEW

**Review by albrja — APPROVED** [review]

File: src/vivarium_gates_child_iv_iron/data/loader.py [hunk 1]
```diff
@@ -67,15 +68,36 @@ def load_population_location(key: str, location: str) -> str:
...
 
 def load_age_bins(key: str, location: str) -> pd.DataFrame:
     return interface.get_age_bins()
 
 
 def load_demographic_dimensions(key: str, location: str) -> pd.DataFrame:
-    return interface.get_demographic_dimensions(location)
+    demographic_cols = ['sex', 'age_start', 'age_end', 'year_start', 'year_end']
+
+    demographic_dimensions = pd.DataFrame([
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/data/loader.py]: Look at CIFF for how to do this.

File: src/vivarium_gates_child_iv_iron/data/loader.py [hunk 2]
```diff
@@ -97,37 +119,6 @@ def load_metadata(key: str, location: str):
     return entity_metadata
 
 
-def load_categorical_paf(key: str, location: str) -> pd.DataFrame:
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/data/loader.py]: Add this back in

File: src/vivarium_gates_child_iv_iron/results_processing/process_results.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import itertools
 from pathlib import Path
-from typing import NamedTuple, List
+from typing import NamedTuple, List, Dict, Union
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/results_processing/process_results.py]: order these alphabetically.

File: src/vivarium_gates_child_iv_iron/tools/make_artifacts.py [hunk 4]
```diff
@@ -35,7 +35,8 @@ def running_from_cluster() -> bool:
 
 
 def check_for_existing(output_dir: Path, location: str, append: bool, replace_keys: Tuple) -> None:
-    existing_artifacts = set([item.stem for item in output_dir.iterdir()
+    # need to explicitly cast to Path from str
+    existing_artifacts = set([item.stem for item in Path(output_dir).iterdir()
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/tools/make_artifacts.py]: Convert this str to path where check_for_existing is being called

**Review by rmudambi — CHANGES_REQUESTED** [review]

**Review by rmudambi — APPROVED** [review]

**Comment by hussain-jafari** [comment]
Model 0 Demography Artifact

    Category: data artifact
    JIRA issue: [MIC-2895](https://jira.ihme.washington.edu/browse/MIC-2895)
    Research reference: [IV Iron - Children under five](https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_iv_iron/child_model/concept_model.html)

Verification and Testing

Created artifacts for South Asia and Sub-Saharan Africa and checked that population structure, demographic dimensions, etc looked good.

**Pull request merged**